packages/dialtone-css/lib/build/less/components/emoji-picker.less

```diff
@@ -38,6 +38,8 @@
     width: auto;
     max-width: calc(var(--dt-size-925) + var(--dt-size-400));
     margin: 0 var(--dt-space-500);
+    // !important to default value to override popover dialog box-sizing: border-box style
+    box-sizing: content-box !important;
```

Comment:
I am questioning whether it's better to use important, or nested css to increase specificity. I think this is okay in this case.

Reply:
This selector `.d-popover__dialog *` it's strictly invasive, due to `*` it is being applied everywhere.

I prefer also the explicit `!important!` rule here. To expose this popover directive.